server/apps/research/serializers/article_serializer.py

```
@@ -100,14 +167,14 @@
                 article.related_articles.set(related_articles)
 
             return article
-        except Exception as e:            
+        except Exception as e:
             raise serializers.ValidationError(f"Error creating article: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)